misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/CharSequenceContainsRegexAssertionsSpec.kt

```diff
@@ -200,3 +200,196 @@ abstract class CharSequenceContainsRegexAssertionsSpec(
         }
     }
 })
+
+abstract class CharSequenceContainsRegexAssertionsSpecRegex(
```

Comment:
no need to create this class, you can reuse `CharSequenceContainsRegexAssertionsSpec` also for regex.

Reply:
In that case, how do I make the RegexSpex compile (since `CharSequenceContainsRegexAssertionsSpec` only accepts `String` versions)?

Reply:
You have to implement containsAtLeastRegex in a way that it full fills the signature the spec requires (so the same as containsAtLeast) but uses the regex version internally and not the string version